cmd/cdi-importer/importer.go

```diff
@@ -68,17 +68,20 @@ func main() {
 
 	glog.V(1).Infoln("begin import process")
 	dso := &importer.DataStreamOptions{
-		Dest:           dest,
-		DataDir:        dataDir,
-		Endpoint:       ep,
-		AccessKey:      acc,
-		SecKey:         sec,
-		Source:         source,
-		ContentType:    contentType,
-		ImageSize:      imageSize,
-		AvailableSpace: util.GetAvailableSpace(common.ImporterVolumePath),
-		CertDir:        certDir,
+		Dest:                dest,
+		DataDir:             dataDir,
+		Endpoint:            ep,
+		AccessKey:           acc,
+		SecKey:              sec,
+		Source:              source,
+		ContentType:         contentType,
+		ImageSize:           imageSize,
+		AvailableSpace:      util.GetAvailableSpace(common.ImporterVolumePath),
+		CertDir:             certDir,
 		InsecureTLS:    insecureTLS,
+		HasScratchAvailable: util.GetAvailableSpace(common.ScratchDataDir) != int64(-1),
```

Comment:
GetAvailableSpace will return -1 if there is an error statting the directory passed in. I suppose you could argue that if we have a 0 sized PVC this will return true for has scratch space available, and with it being 0 sized we don't have any. But I wrote this with the assumption, if the pvc is not mounted, I don't have scratch space, and thus statting the path will return -1, in which case this is correct.

Reply:
`> 0` makes more sense to me

Reply:
I am not married to the != -1, the result is the same.